wdl-doc/src/struct.rs

```diff
@@ -0,0 +1,58 @@
+//! Create HTML documentation for WDL structs.
+
+use std::fmt::Display;
+
+use html::content;
+use html::text_content;
+use wdl_ast::AstToken;
+// use wdl_ast::v1::MetadataValue;
+use wdl_ast::v1::StructDefinition;
+
+/// A struct in a WDL document.
+#[derive(Debug)]
+pub struct Struct {
+    /// The AST definition of the struct.
+    def: StructDefinition,
+    // /// Any meta entries associated with the struct.
+    // meta: Vec<MetadataValue>,
+}
+
+impl Struct {
+    /// Create a new struct.
+    pub fn new(def: StructDefinition) -> Self {
```

Comment:
This needs to be deleted.

Reply:
What's the issue here? @claymcleod

Reply:
I think this might have overlapped with my comment at the same location, but Clay submitted his right when you submitted the fix to remove the commented out code.

Reply:
Ok. In that case I'll merge as soon as the CI turns Green 🎉